packages/common/mui/src/lib/data-table/data-table.tsx

```diff
@@ -193,6 +203,54 @@ export const DataTable = ({
       {tabBarProps && <DataTableTabBar tabBarProps={tabBarProps} />}
 
       {TabComponentVisible && CustomTabComponent && <CustomTabComponent />}
+      {pendingTokenData && pendingTokenData.pendingTokens.length > 0 && (
+        <Box>
+          <ErrorMessage
+            message={
+              pendingMessage || (
+                <FormattedMessage defaultMessage={'action pending...'} />
```

Comment:
Do you need to have a default message here? It looks like you can pass an undefined prop here.

Reply:
Can we pull this into a separate component? This file is getting pretty big